src/main/java/com/amazon/opendistroforelasticsearch/security/dlic/rest/api/MigrateApiAction.java

```diff
@@ -119,15 +122,26 @@ protected void handlePost(RestChannel channel, RestRequest request, Client clien
         final SecurityDynamicConfiguration<WhitelistingSettings> whitelistingSettingV6 = (SecurityDynamicConfiguration<WhitelistingSettings>) load(CType.WHITELIST, true);
         final SecurityDynamicConfiguration<AuditConfig> auditConfigV6 = (SecurityDynamicConfiguration<AuditConfig>) load(CType.AUDIT, true);
 
+        final ImmutableMap.Builder<String, SecurityDynamicConfiguration<?>> builder = ImmutableMap.builder();
+
         final SecurityDynamicConfiguration<ActionGroupsV7> actionGroupsV7 = Migration.migrateActionGroups(actionGroupsV6);
+        builder.put(CType.ACTIONGROUPS.toLCString(), actionGroupsV7);
```

Comment:
Nit: An alternative approach would be storing objects in a set.
The SecurityDynamicConfiguration<> objects have a method called getCType() that will give you CType of an object.